mantidproject/mantid
# Indirect - Data Analysis - TransformToIqt - Replace NaN values with zeroes

NaN values produced during the TransformIqt algorithm (found in the last bin of the output workspace) are currently not being replaced with zero values.

This PR applys the ReplaceSpecialValues algorithm to replace the NaN values with zero values.

**To test:**
1. Navigate to Indirect > Data Analysis > I(Q, t)
2. In the sample field provide a reduced (\*\_red) file (provided below)
3. In the resolution field provide a resolution (\*\_res) file (provided below)
4. Run the algorithm
5. Open the output workspace and check there are no NaN values contained within it

Reduced and Resolution file:
[IQT.zip](https://github.com/mantidproject/mantid/files/1281862/IQT.zip)



Fixes #20396. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
